tools/extra/qpafilter/qpafilter.py

```diff
@@ -104,7 +104,8 @@ def filter(self, items):
                 warning = (override * critical) / 100.0
             else:
                 override = 0.0
-                warning = (critical * self.args.warn_temp) / self.args.crit_temp
+                warning = ((critical * self.args.virt_warn_temp) /
+                          self.args.virt_fatal_temp)
 
```

Comment:
are we doing that 0.5 granularity  calculation here or somewhere else?
For example, if the temp is 80, then we pass the BMC, 160... vs. if the temp is 80.5, we pass to BMC 161?

Reply:
Not doing that yet.  I'll add it once we get an updated input file format.